src/modules/ProxiedModuleApi.ts

```diff
@@ -144,16 +144,13 @@ export class ProxiedModuleApi implements ModuleApi {
      * @override
      */
     public async overwriteAccountAuth(accountInfo: AccountAuthInfo): Promise<void> {
-        dispatcher.dispatch<OverwriteLoginPayload>(
+        await doSetLoggedIn(
```

Comment:
Seems like this was the only caller which used `OverwriteLoginPayload` so it should be cleaned up

Reply:
thanks, okay, I have removed `OverwriteLoginPayload` and it's usages from the code.

Reply:
This change actually breaks a lot of tests (approximately 10%). It looks not clear to me. It seems that some mocking is not working properly anymore. Is it valid to use `Lifecycle` this way? If yes maybe there is some idea or hint how the tests could be fixed? @t3chguy could you please help?

Reply:
Given the method you are calling was not previously exported - it wasn't a designed use for it to be called like this.

As for the test failures, would need to step through the code to see what is wrong. Not something I'm able to dive into right now unfortunately